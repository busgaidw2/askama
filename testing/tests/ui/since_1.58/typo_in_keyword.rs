../before_1.58/typo_in_keyword.rs