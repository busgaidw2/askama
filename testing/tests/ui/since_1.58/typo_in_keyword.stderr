error: problems parsing template source at row 1, column 26 near:
       "endfo%}\n12345678901234567890123456789012"...
 --> tests/ui/since_1.58/typo_in_keyword.rs:3:10
  |
3 | #[derive(Template)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `Template` (in Nightly builds, run with -Z macro-backtrace for more info)
